cmd/kueuectl/app/util/helpers.go

```diff
@@ -1,38 +0,0 @@
-/*
-Copyright 2024 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package util
-
-import (
-	"github.com/spf13/cobra"
-	"k8s.io/klog/v2"
-)
-
-func FlagString(cmd *cobra.Command, flag string) string {
-	s, err := cmd.Flags().GetString(flag)
-	if err != nil {
-		klog.Fatalf("error accessing flag %s for command %s: %v", flag, cmd.Name(), err)
```

Comment:
Is the error coming from `cmd.Flags().GetStringArray` containing the information about the flag & command?

Reply:
Thank you for review @mimowo. 

Flag information should be in. https://github.com/kubernetes/kubernetes/blob/297570e06a88db23e16dbdbf6ce3173fe0ae376c/vendor/github.com/spf13/pflag/string_array.go#L72 calls https://github.com/kubernetes/kubernetes/blob/297570e06a88db23e16dbdbf6ce3173fe0ae376c/vendor/github.com/spf13/pflag/flag.go#L376-L390 but I'm not sure about the cmd. 

However, since the caller already knows the command name, flag should be more beneficial information.

Reply:
Thanks for checking, I agree there is no much point repeating the command to the caller. Unless I'm missing something, cc @mbobrovskyi